packages/ui/src/modal-side/ModalSide.css

```diff
@@ -86,33 +78,38 @@
   align-items: center;
 }
 
-.ui-ModalSide--SectionGlossaryContentLine {
-  display: grid;
-  grid-template-columns: repeat(5, 1fr);
-  align-items: center;
+.ui-ModalSide--SectionNumber,
+.ui-ModalSide--SectionNumberLarge {
+  grid-column: span 1;
+  text-align: left;
+  padding: var(--layout-padding-medium);
+  padding-right: var(--modal-side-number-padding);
+  font-weight: var(--typography-font-weights-bold);
 }
 
-.ui-ModalSide--SectionBuildingCodeContentLine {
-  display: grid;
-  padding: var(--layout-padding-medium);
-  align-items: center;
+.ui-ModalSide--SectionNumber {
+  font-size: var(--typography-font-size-h4);
 }
 
-.ui-ModalSide--BuildingCodeList {
-  padding: revert;
+.ui-ModalSide--SectionNumberLarge {
+  font-size: var(--typography-font-size-h3);
 }
 
-.ui-ModalSide--SectionNumber {
-  grid-column: span 1;
-  text-align: center;
-  font-size: var(--typography-font-size-h5);
+.ui-ModalSide--SectionHeader,
+.ui-ModalSide--SectionHeaderLarge {
```

Comment:
I think we need a little style work on the mobile version. Some things I'm noticing:

1. Spacing with outside edge is too large.
2. Font size and line height of bold headers is too large
3. In the screenshot below you can see the Article 9.9.9.1 header is actually indented from the Subsection 9.9.9 header.

![Screenshot 2024-07-08 at 1 19 01 PM](https://github.com/bcgov/House-Innovations-HOUSNAV/assets/41478918/d8cd72bb-784b-4fa9-8f87-d93f5e253453)

Reply:
Some of the same things apply to desktop as well but I also think desktop _**looks**_ fine so I'm cool with leaving this as is.

![Screenshot 2024-07-08 at 1 25 06 PM](https://github.com/bcgov/House-Innovations-HOUSNAV/assets/41478918/e3d1b0fb-2a9f-4919-a9e1-37b7470f1d3d)

Reply:
Yeah one of the designs show H3/H4 for the headers (Which I have) and the other shows H4/H5 which is slightly smaller. Which is the one I had before but when I checked I saw the larger ones so I updated it lol. So which one is right? I'm thinking the smaller one is better as you suggested?

Reply:
From what I can tell It's a responsive thing. At desktop they use the H3/H4 styles and on mobile they use H4/H5. That's what the responsive breakdown [Templates page](https://www.figma.com/design/fABseEsz7QvzaUXz9lRyPw/BCBC---Design-Discovery?node-id=770-7499&t=bsI0kbxLlgQMEgMq-4) shows